### DISCUSSION TITLE
The operationId is duplicated？？why?

### DISCUSSION THREAD

**Discussion by oimmc** [acacode/swagger-typescript-api#1136] [Q&A]
![image](https://user-images.githubusercontent.com/18304330/187588722-4f46333b-a892-450c-aa93-0b0c3c4c3736.png)

// v3/api-docs .json
operationId: delete1,
![image](https://user-images.githubusercontent.com/18304330/187589316-51d0d634-557a-441c-8048-913eac160ab0.png)

I want operationId: delete

or
![image](https://user-images.githubusercontent.com/18304330/187589016-207a5308-33a7-4086-83a3-835e5edbed2d.png)

**Comment by js2me** [comment]
Hi @oimmc can you share part of swagger schema with `delete` operations ?

**Comment by Amar97** [comment]
In essence this means your api spec is not completely correct. We have the same issue because we are generating some functions with generics, which causes them to have the same name and operationID. OperationID should be unique, which is why most api spec generators will add a suffix like _1 or _2. 

I haven't really figured out how to solve it in Spring Boot so maybe it would be nice to have the option here to ignore the suffixes.
